running.md

```diff
@@ -160,19 +160,25 @@ The standard Fedora CoreOS image does not contain Cockpit packages.
 
 2. Reboot
 
-3. Run the Cockpit web service with this privileged container (as root):
+This is sufficient if the CoreOS machine will only be connected to from another host running Cockpit. If you also want to run a web server for logging in directly:
+
+1. Enable password based SSH logins, unless you only use [SSO logins](https://cockpit-project.org/guide/latest/sso.html):
+   ```
+   echo 'PasswordAuthentication yes' | sudo tee /etc/ssh/sshd_config.d/02-enable-passwords.conf
+   sudo systemctl try-restart sshd
+   ```
+
+2. Run the Cockpit web service with this privileged container (as root):
    ```
    podman container runlabel --name cockpit-ws RUN docker.io/cockpit/ws
    ```
 
-4. Make Cockpit start on boot:
+3. Make Cockpit start on boot:
```

Comment:
I _think_ we can use 3. 4. 5. here still.

Reply:
That's not how markdown enumerations work, though. You can also write "0." or "1." everywhere, and it will always do the "proper" counting from 1. I'll try, though.

Reply:
Yup, pushed, and it's still rendered as "1. 2. 3.". Doesn't hurt though to keep it as 3. 4. 5. in the MD sources.